[Docs] Update Sharing docs (#190318)
elastic/kibana: docs/user/reporting/index.asciidoc

:frontmatter-description: {kib} provides you with several options to share *Discover* saved searches, dashboards, *Visualize Library* visualizations, and *Canvas* workpads with others, or on a website.
:frontmatter-tags-products: [kibana]

{kib} provides you with several options to share *Discover* saved searches, dashboards, *Visualize Library* visualizations, and *Canvas* workpads. These sharing options are available from the *Share* menu in the toolbar.

[float]
[[share-a-direct-link]]
== Share with a direct link

> You can share direct links to saved searches, dashboards, and visualizations. When clicking **Share**, look for the **Links** tab to get the shareable link and copy it. When sharing an object with unsaved changes, you're getting a temporary link that might break in the future for example in case of upgrade. Save the object to get a permanent link instead.

Author: ```suggestion
You can share direct links to saved searches, dashboards, and visualizations. When clicking **Share**, look for the **Links** tab to get the shareable link and copy it. 

TIP: When sharing an object with unsaved changes, you get a temporary link that might break in the future, for example in case of upgrade. Save the object to get a permanent link instead.
```

To access the object shared with the link, users need to authenticate.

> TIP: Anonymous users can also access the link if you have configured <<anonymous-authentication,Anonymous authentication>> and your anonymous service account has privileges to access what you want to share.

Author: ```suggestion
Anonymous users can also access the link if you have configured <<anonymous-authentication,Anonymous authentication>> and your anonymous service account has privileges to access what you want to share.
```

:frontmatter-description: {kib} provides you with several options to share *Discover* saved searches, dashboards, *Visualize Library* visualizations, and *Canvas* workpads with others, or on a website.
:frontmatter-tags-products: [kibana]

{kib} provides you with several options to share *Discover* saved searches, dashboards, *Visualize Library* visualizations, and *Canvas* workpads. These sharing options are available from the *Share* menu in the toolbar.

[float]
[[share-a-direct-link]]
== Share with a direct link

You can share direct links to saved searches, dashboards, and visualizations. When clicking **Share**, look for the **Links** tab to get the shareable link and copy it. 

TIP: When sharing an object with unsaved changes, you get a temporary link that might break in the future, for example in case of upgrade. Save the object to get a permanent link instead.

To access the object shared with the link, users need to authenticate.

Anonymous users can also access the link if you have configured <<anonymous-authentication,Anonymous authentication>> and your anonymous service account has privileges to access what you want to share.

> image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt49f2b5a80ec89a34/66b9e919af508f4ac182c194/share-dashboard.gif[getting a shareable link for a dashboard]

Reviewer: Is this a temporary image URL?

Author: No, it's a link to a gif hosted on our asset service (contentstack). We put gifs there to avoid adding too heavy assets to the repo.

. Click *Share > Embed code*.

. Specify which parts of the dashboard you want to include: Top menu, query, time filter, and filter bar.

. Click *Copy embed code*.

//* To automatically log in anonymous users when you have multiple authentication providers enabled, select *Public URL*.
//+

> //NOTE: *Public URL* is available only when anonymous access is configured and your anonymous service account has privileges to access what you want to embed. For more information, refer to <<anonymous-authentication,Anonymous authentication>>.

Reviewer: These lines can be removed

Author: Ok Awesome, I commented them out in case they were still needed, but I'll remove them altogether. Thanks!

Author: ```suggestion

```
